vignettes/varying_susceptibility.Rmd

```diff
@@ -252,8 +270,16 @@ This example model considers half of each age group to be in the immunised and n
 
 In general terms, this could be interpreted as the rate of vaccine uptake, or as the effect of existing immunity from previous infection by a similar pathogen.
 
+::: {.alert .alert-secondary}
+
+### Getting data on within-group susceptibility differences {-}
+
+For within-group differences in susceptibility, such as due to immunisation, users could obtain this information from [age-specific vaccination uptake statistics from national governments (here, the UK)](https://coronavirus.data.gov.uk/details/vaccinations?areaType=nation%26areaName=England#card-vaccination_uptake_by_vaccination_date_age_demographics).
+When antibody response decays are known for an immunisation course [@iyer2020], more detailed statistics on the [percentage of people vaccinated (here, in the UK)](https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/conditionsanddiseases/articles/coronaviruscovid19latestinsights/vaccines) in the preceding few months only, may be more informative.
```

Comment:
I suggest removing this added paragraph. This situation is not linked with any applied case scenario. I prefer to wait to have more published case studies to cite in the documentation.

Reply:
I think it's a good consideration actually, which is relevant to future P3 work on vaccination schedules, and the reference can be kept, but perhaps moved to the primer.

I can imagine that if some reduction in antibody titres is known after certain periods (e.g. vaccination or natural infection), this could indeed be modelled as somewhat different susceptibility for the part of the population which was immunised earlier.

Reply:
I do agree, and good to know about its relevance! Then let's keep it here due to the topic. Is much more specific to vaccine mediated immunity than to the primer.

Reply:
As an alternative, to highlight that core idea then, we can include the `in the preceding few months only` to the linked text:

`(...) the [percentage of people vaccinated in the preceding few months only (here, in the UK)](https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/conditionsanddiseases/articles/coronaviruscovid19latestinsights/vaccines), (...)`

Reply:
Fixed in https://github.com/epiverse-trace/finalsize/pull/121/commits/e40a3fcb4682a3c2596ed1761df8bdf4dc72dd20